test/adapters/IDB_test.js

```diff
@@ -474,8 +493,11 @@ describe("adapter.IDB", () => {
     });
   });
 
-  /** @test {IDB#loadDump} */
-  describe("#loadDump", () => {
+  /** 
+   * @deprecated
+   * @test {IDB#loadDump} 
+   */
+  describe("Deprecated #loadDump", () => {
```

Comment:
I don't think we need the old tests for `loadDump`. Just add a test that it calls `importBulk`.

Reply:
I'm not exactly sure how to test that `loadDump` is calling `importBulk`.  Do you have any suggestions on how to do that?

Reply:
Ok, I figured out that I have to use `sinon.spy()` or one of its variations.  Now trying to figure out how to do that...